rental.rb

```diff
@@ -0,0 +1,18 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date)
+    @date = date
+  end
+
+  def book=(book)
+    @book = book
+    book.rentals.push(self) unless book.rentals.include?(self)
+  end
+
+  def person=(person)
+    @person = person
+    person.rentals.push(self) unless person.rentals.include?(self)
+  end
+end
```

Comment:
- [ ] Instead of defining a method for book and person, You can simply add them inside the initialize method. with this, the rental will be adding itself to the @book and @person achieving the `belongs-to` association 

e.g

```ruby
def initialize(date, book, person)
    .
    .
    .
    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
```